diff --git a/index.d.ts b/index.d.ts
index c94bb768b3bec12657a413fa278515abe05c05a9..adcf168554583d32384a8ff7e679d719cbf711a1 100644
--- a/index.d.ts
+++ b/index.d.ts
@@ -9,7 +9,7 @@ import * as superagent from "superagent";

 export = supertest;

-declare function supertest(app: any): supertest.SuperTest<supertest.Test>;
+declare function supertest(app: any, options?: supertest.Options): supertest.SuperTest<supertest.Test>;
 declare namespace supertest {
     interface Response extends superagent.Response {}

@@ -31,7 +31,11 @@ declare namespace supertest {
         end(callback?: CallbackHandler): this;
     }

+    interface Options {
+        http2?: boolean;
+    }
+
-    interface AgentOptions {
+    interface AgentOptions extends Options {
         ca?: any;
     }
     function agent(app?: any, options?: AgentOptions): SuperAgentTest;
